pkg/capicluster/cluster.go

```diff
@@ -14,12 +18,24 @@ type Object struct {
 	Options loggedcluster.Options
 }
 
-func (o Object) GetLoggingLabel() string {
+func (o Object) HasLoggingEnabled() bool {
 	labels := o.Object.GetLabels()
 
-	value := labels[key.LoggingLabel]
+	// If logging is disabled at the installation level, we return false
+	if !o.Options.EnableLoggingFlag {
+		return false
+	}
 
-	return value
+	loggingLabelValue, ok := labels[key.LoggingLabel]
+	if !ok {
+		return true
+	}
+
+	loggingEnabled, err := strconv.ParseBool(loggingLabelValue)
+	if err != nil {
+		return false
+	}
```

Comment:
If there is no label, it returns `true` => this means logging becomes enabled by default, right?
But if there's an error in `ParseBool`, it returns `false` => why not sticking to the default behaviour (`true`) then?

Reply:
Makes sense

Reply:
Fixed